tests/test_config.py

```diff
@@ -188,6 +244,16 @@ def testMultipleConfigFiles(self):
 password=password
 owner=root
 env=staging
+rhsm_username=rhsm_admin1
+rhsm_password=rhsm_password1
+rhsm_hostname=host1
+rhsm_ssl_port=12341
+rhsm_prefix=prefix1
+rhsm_proxy_hostname=proxyhost1
+rhsm_proxy_port=43211
+rhsm_proxy_user=proxyuser1
+rhsm_proxy_password=proxypass1
+rhsm_insecure=1
```

Comment:
Just want to check, does this function ok if you don't specify all of these properties? I.e. do we fallback to the original rhsm.conf values if something isn't found?